### DISCUSSION TITLE
how to set auto connect when restart my phone

### DISCUSSION THREAD

**Discussion by FxamarAboali** [schwabe/ics-openvpn#1717] [Q&A]
how to set my config auto connect when i restart my phone

**Comment by schwabe** [comment]
Use Android's own Always on VPN feature under VPN settings in the Android system settings.

**Reply by FxamarAboali** [reply]
I cant do it . can u explain how ,by image please

**Reply by schwabe** [reply]
See https://support.google.com/android/answer/9089766?sjid=8897381936222863189-EU
